pdb-alignment-pipeline/src/main/java/org/cbioportal/pdb_annotation/util/ReadConfig.java

```diff
@@ -0,0 +1,165 @@
+package org.cbioportal.pdb_annotation.util;
+
+import java.io.File;
+import java.util.*;
+
+import org.apache.commons.io.FileUtils;
+
+public class ReadConfig {
+	
+    public static String makeblastdb;
+    public static String blastp;
+    public static String workspace;
+    public static String resource_dir;
+    public static String tmpdir;
+    public static String pdb_seqres_download_file;
+    public static String pdb_seqres_fasta_file;
+    public static String ensembl_download_file;
+    public static String ensembl_fasta_file;
+    public static String sql_insert_file;
+    public static String sql_delete_file;
+    public static String blast_para_evalue;
+    public static String blast_para_threads;
+    public static String ensembl_input_interval;
+    public static String sql_insert_output_interval;
+    public static String mysql;
+    public static String username;
+    public static String password;
+    public static String db_schema;
+    public static String db_schema_script;
+    
+    public static String pdbwholeSource;
+    public static String ensemblwholeSource;
+    
+    public static String updateTxt;
+    public static String updateFasta;
+    public static String delPDB;
+    
+    public static String updateDAY_OF_WEEK;
+    public static String updateHOUR_OF_DAY;
+    public static String updateMINUTE;
+    public static String updateSECOND;
+    public static String updateMILLISECOND;
+    public static String updateDELAY;
+    
+    public static String sql_ensemblSQL;
+
+	
+	public ReadConfig(){
+		try{
+			HashMap hm = new HashMap();
+			File file = new File("src/main/resources/application.properties");
```

Comment:
Thanks, I will try to digest and modify these codes

On Mon, Jul 18, 2016 at 12:14 PM, onursumer <notifications@github.com>
wrote:

> In
> pdb-alignment-pipeline/src/main/java/org/cbioportal/pdb_annotation/util/ReadConfig.java
> <https://github.com/cBioPortal/pdb-annotation/pull/12#discussion_r71188448>
> :
>
> > +    public static String delPDB;
> > +
> > +    public static String updateDAY_OF_WEEK;
> > +    public static String updateHOUR_OF_DAY;
> > +    public static String updateMINUTE;
> > +    public static String updateSECOND;
> > +    public static String updateMILLISECOND;
> > +    public static String updateDELAY;
> > +
> > +    public static String sql_ensemblSQL;
> > +
> > +	
> > +	public ReadConfig(){
> > +		try{
> > +			HashMap hm = new HashMap();
> > +			File file = new File("src/main/resources/application.properties");
>
> @juexinwang <https://github.com/juexinwang> You don't need to manually
> read the properties files, and set the values. The code below is an example
> of automatic mapping of a property to a class variable:
>
> private String headerServiceURL;
> @Value("${pdb.header_service_url}")public void setHeaderServiceURL(String headerServiceURL)
> {
>     this.headerServiceURL = headerServiceURL;
> }
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cBioPortal/pdb-annotation/pull/12/files/732f3ee88338f2eef75f0e262929715c5e2d7a6f#r71188448>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADL7zEMC5ynwqP-bw_ax_nTOrBBZXPnbks5qW7R0gaJpZM4I5V0q>
> .
>